sync-branches/action.yaml

```
@@ -64,6 +64,13 @@ runs:
       with:
         git-cliff-args: origin/${{ inputs.base-branch }}..HEAD
 
+    - name: Replace PR Number to URL
+      id: replace-pr-number-to-url
+      run: |
+        r=$(echo ${{ steps.generate-changelog.outputs.changelog }} | sed -r "s|\((#[0-9]+)\)|($ {{ inputs.sync-target-repository }}/pull/\1)|g")
+        echo ::set-output name=changelog::$r
```

Please use `env` to escape special characters like this.
Related: https://github.com/autowarefoundation/autoware-github-actions/issues/82

```suggestion
        echo ::set-output name=changelog::$(echo "$CHANGELOG" | sed -r "s|\((#[0-9]+)\)|($ {{ inputs.sync-target-repository }}/pull/\1)|g")
      env:
        CHANGELOG: ${{ steps.generate-changelog.outputs.changelog }}
```